codegen_glibmm/templates/stub.h.templ

```diff
@@ -26,7 +26,7 @@ public:
     void unregister_object();
 
     int usage_count() const {
-        return m_registeredObjectId == 0 ? 0 : 1;
+        return m_registered_objects.size();
```

Comment:
Maybe we should change return type of usage_count() to std::size_t? (Will never be a problem in practice with an int, but thinking that e.g. linters might complain about signed vs unsigned and size.)

Reply:
Nah, I wouldn't use `size_t` for anything but memory ranges and offsets. If we get issues, we'll fix them with a cast.

I'm not against changing the return type to `unsigned int`, though, if you prefer that.

Reply:
Sure, and a `static_cast<unsigned int>()` as well then I guess to not get warnings about truncation.